digits/model/images/forms.py

```diff
@@ -34,3 +30,81 @@ class ImageModelForm(ModelForm):
             tooltip = "Subtract the mean file or mean pixel for this dataset from each image."
             )
 
+    aug_flip = utils.forms.SelectField('Flipping',
+            choices = [
+                ('none', 'None'),
+                ('fliplr', 'Horizontal'),
+                ('flipud', 'Vertical'),
+                ('fliplrud', 'Horizontal and/or Vertical'),
+                ],
+            default='none',
+            tooltip = "Randomly flips each image during batch preprocessing."
+            )
+
+    aug_quad_rot = utils.forms.SelectField('Quadrilateral Rotation',
+            choices = [
+                ('none', 'None'),
+                ('rot90', '0, 90 or 270 degrees'),
+                ('rot180', '0 or 180 degrees'),
+                ('rotall', '0, 90, 180 or 270 degrees.'),
+                ],
+            default='none',
+            tooltip = "Randomly rotates (90 degree steps) each image during batch preprocessing."
+            )
+
+    aug_rot_use = utils.forms.BooleanField('Arbitrary Rotation',
+            default = False,
+            tooltip = "You can augment your dataset by performing rotation on images.",
+            validators=[ 
+                ]
+            )
+    aug_rot = utils.forms.IntegerField('Rotation (+- deg)',
+            default=45,
+            validators=[ 
+                validators.NumberRange(min=0, max=360)
+                ],
+            tooltip = "The uniform-random rotation angle that will be performed during batch preprocessing."
+            )
+
+    aug_scale_use = utils.forms.BooleanField('Random Scaling',
```

Comment:
Yes we can. Do you want me to? I was following the redundant convention of the crop and croplen (in main.lua, see below), but extended it in this case to the frontend.
```
-k,--crop (default no) If this option is 'yes', all the images are randomly cropped into square image. And croplength is provided as --croplen parameter
-l,--croplen (default 0) crop length. This is required parameter when crop option is provided
```